delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/indexing/ViewLens.scala

```diff
@@ -0,0 +1,27 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.indexing
+
+import ch.epfl.bluebrain.nexus.sourcing.projections.ProjectionId.ViewProjectionId
+
+import java.util.UUID
+
+/**
+  * A lens on a View
+  * @tparam A the view type
+  */
+trait ViewLens[A] {
```

Comment:
Should not we just have a trait defined in the sdk that would be needed to be implemented by the different views in the different plugins rather than having too defined a Lens ?
Plus we only need basic information here

Reply:
I didn't want to couple those things _yet_.
Maybe we end up doing it this way, maybe not.